src/style.css

```diff
@@ -0,0 +1,63 @@
+/* Reset and base styles  */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul,
+li {
+  display: flex;
```

Comment:
You don't need to specify additional styles props for tags where you use classes directly, class selectors are more specific, so this definition doesn't make sense